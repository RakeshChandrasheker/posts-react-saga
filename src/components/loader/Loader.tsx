import { FunctionComponent } from 'react';
import './Loader.scss';
import Spinner from '../../assets/icon/spinner.svg';

const Loader: FunctionComponent = (): JSX.Element => {
  return (
    <div className="Loader">
      <img src={Spinner} alt={Spinner} />
    </div>
  );
};

export default Loader;