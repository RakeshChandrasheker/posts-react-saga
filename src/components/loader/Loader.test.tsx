import { shallow } from 'enzyme';
import Loader from './Loader';

describe('<Loader />', () => {

  const wrapper = shallow(<Loader />);

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should Match Snapshot', () => {
    expect(wrapper.getElement()).toMatchSnapshot();
  });

  afterAll(() => {
    wrapper.unmount();
  });
});
