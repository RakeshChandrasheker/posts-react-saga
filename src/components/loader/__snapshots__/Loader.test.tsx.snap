// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should Match Snapshot 1`] = `
<div
  className="Loader"
>
  <img
    alt="spinner.svg"
    src="spinner.svg"
  />
</div>
`;
